cli/cmd/common.go

```diff
@@ -22,3 +23,13 @@ func handleCmdErr(cmd *cobra.Command, err error) {
 		log.Fatalf(err.Error())
 	}
 }
+
+var errNoConfig = errors.New("tt.yaml not found, you need to create tt" +
+	" environment config with tt init")
```

Comment:
```suggestion
	" environment config with 'tt init'")
```
up to you, i think it looks better to highlight commands in (').

Reply:
Thank you, updated.